### PR TITLE
Login

### PR DESCRIPTION
**Pull request by dmfarcas**
Dupa cum am vorbit la hackathonul de pe 23, vom muta sectiunea de admin in acest proiect pentru a avea consistenta in componente. Dupa cum e folosita aplicatia, e de inteles ca e ok sa fie un singur codebase.

### CODE REVIEW

File: app/components/Menu/index.js [hunk 1]
```diff
@@ -60,28 +59,44 @@ export default class Menu extends React.PureComponent {
     const path = this.props.pathname.split('/');
     const getLocation = path[1] || path[0];
 
-    if (this.props.pathname.length > 1 && this.props.pathname !== '/multumim') {
-      document.getElementById(getLocation).parentNode.querySelector('.selected').classList.remove('selected');
-      document.getElementById(getLocation).firstElementChild.classList.add('selected');
-    } else {
-      // document.querySelector('#menu-list .selected').classList.remove('selected');
-    }
+    // if (this.props.pathname.length > 1 && this.props.pathname !== '/multumim') {
```
Comment (danichim) [inline app/components/Menu/index.js]: Get class selected without dom manipulation :1st_place_medal:

Comment (dmfarcas) [reply app/components/Menu/index.js]: On it :P

File: = [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (dmfarcas) [inline =]: remove this file

File: app/api/baseApi.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import config from './config';
```
Comment (dmfarcas) [inline app/api/baseApi.js]: might not be the right place for a config file

File: app/containers/Acasa/MainContainer/components/LeftContainer/index.js [hunk 4]
```diff
@@ -100,100 +128,118 @@ export class LeftContainer extends React.PureComponent {
...
+      this.state.description &&
+      this.props.incidentId &&
+      this.props.token !== undefined;
     return (
       <StickyContainer className="col-xs-12 col-lg-7 form-col">
         <Interact className="interact">
           <Sticky isActive={this.shouldBeSticky()}>
             <h2>Adaugă o sesizare</h2>
-            <p>Mai jos găsești formularul prin care poți să trimiți o sesizare. Dacă nu ești sigur ce poate fi subiectul unei sesizări, verifică aici <Link to="reguli-vot">Regulile votului</Link>. <small><span style={{ fontWeight: 700 }}>Important:</span> sesizările trimise vor fi promovate în scop informativ în social media și către mass-media. Dacă dorești un răspuns direct de la autorități, te rugăm să te adresezi Biroului Electoral de Circumscripție sau Biroului Electoral Central. Mai multe detalii <Link to="despre-noi">aici.</Link></small></p>
+            <p>
```
Comment (dmfarcas) [inline app/containers/Acasa/MainContainer/components/LeftContainer/index.js]: not sure what happened here. might as well checkout this file

File: app/containers/Admin/index.js [hunk 5]
```diff
@@ -28,10 +29,13 @@ export default class Admin extends React.PureComponent { // eslint-disable-line
               <h1>Admin</h1>
             </div>
             <Content>
+              x
```
Comment (dmfarcas) [inline app/containers/Admin/index.js]: oops.

**Comment by dmfarcas** [comment]
Nu am putut pune label, dar e ready for review 👯

**Pull request merged**